spec/use_cases/administrator/generate_radius_secret_key_spec.rb

```diff
@@ -1,5 +1,7 @@
 describe UseCases::Administrator::GenerateRadiusSecretKey do
+  subject(:use_case) { described_class.new }
+
   it 'has a length of 20 characters' do
-    expect(subject.execute.length).to eq(20)
+    expect(use_case.execute.length).to eq(20)
```

Comment:
Never seen this one before.  The linter prefers that you not use subject directly?

Reply:
Yeah you have to give the subject a name.